DotNetCasClient/Validation/CasSaml11Response.cs

```diff
@@ -98,7 +98,7 @@ private void ProcessValidAssertion()
         {
             protoLogger.Debug("Unmarshalling SAML response");
             XmlDocument document = new XmlDocument();
-            document.Load(new StringReader(_CasResponse));
+            document.LoadXml(_CasResponse);

```

Comment:
Do you happen to have a CAS server set up and tested this change?  I'm wondering if there might be a BOM issue.... didn't test this yet though.